scripts/setup.sh

```
@@ -3,10 +3,28 @@
 VERBOSE=false
-for arg in "$@"; do
-  [[ "$arg" == "--verbose" || "$arg" == "-v" ]] && VERBOSE=true
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+  -v | --verbose) VERBOSE=true ;;
+  -y | --yes) YES_OVERRIDE=true ;;
+  -s | --select)
+    SELECT_OVERRIDE="$2"
+    shift
+    ;;
+  --select=*) SELECT_OVERRIDE="${1#*=}" ;;
```

Shellcheck (0.11.0)

[warning] SELECT_OVERRIDE appears unused. Verify use (or export if used externally).

(SC2034)